manifests/0000_05_serviceservingcertsigner_05_deploy.yaml

```diff
@@ -19,7 +19,7 @@ spec:
       serviceAccountName: openshift-service-cert-signer-operator
       containers:
       - name: operator
-        image: openshift/origin-service-serving-cert-signer:latest
+        image: quay.io/openshift/origin-service-serving-cert-signer:latest
```

Comment:
Your filenames need to be dash separated - `0000_05_service-serving-cert-signer_05_deploy.yaml`.  That makes it consistent with everyone elses.